lib/jekyll/tags/link.rb

```diff
@@ -34,4 +35,4 @@ def render(context)
   end
 end
 
-Liquid::Template.register_tag(Jekyll::Tags::Link.tag_name, Jekyll::Tags::Link)
```

Comment:
this is a method call just to output a simple string.. and that method actually chains 4 other methods to `self` (another call)..
```ruby
def tag_name
  self.name.split("::").last.downcase
end
```